src/inc/corcompile.h

```diff
@@ -1863,6 +1863,8 @@ class ICorCompileInfo
     virtual void EncodeTypeLayout(CORINFO_CLASS_HANDLE classHandle, SigBuilder * pSigBuilder) = 0;
 
     virtual BOOL AreAllClassesFullyLoaded(CORINFO_MODULE_HANDLE moduleHandle) = 0;
+
+    virtual int GetStableMethodHashCode(CORINFO_METHOD_HANDLE hMethod) = 0;
```

Comment:
Call this `GetVersionResilientMethodHashCode` to make it distinct from the existing stable hashcode?